docker/Dockerfile

```diff
@@ -14,6 +14,7 @@ COPY . .
 RUN cargo chef prepare --recipe-path recipe.json
 
 FROM chef AS builder
+SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
```

Comment:
Nice, `-x -o pipefail` might be settings we want to apply to all our bash invocations.
(As we change them over time.)